lib/wpc_proto/internal_modules/proto_otap.c

```diff
@@ -44,7 +44,7 @@ bool Proto_otap_init(void)
 
 void Proto_otap_close(void)
 {
-    WPC_unregister_for_data();
```

Comment:
Ok then my comment on commit 1 is answered :-) 
Please amend the commit 1 then so that it doesn't appear anyway.
For commits are atomic so good to keep them independent (no squash)

Reply:
Ok, I did the amend, and rebase the other commits. But I still think we could squash the whole scratchpad branch (once we solved the details).